packages/fxa-settings/src/components/Nav/index.tsx

```diff
@@ -6,7 +6,7 @@ import React from 'react';
 import classNames from 'classnames';
 import LinkExternal from 'fxa-react/components/LinkExternal';
 import { ReactComponent as OpenExternal } from './open-external.svg';
-import { useAccount } from 'fxa-settings/src/models';
+import { useAccount } from '../../models';
```

Comment:
I've been mostly seeing absolute paths for things within the package you are working it, I think it makes sense for those, as it shows you that the file is within the same package at a glance, but I would be happy either way, so long as it's consistent.

Reply:
tbh i think the fxa-settings one was an auto-import (mistake) from vscode. we should use relative paths "../" since it plays nicer with both ts and compiled js